x/dispensation/client/cli/query.go

```diff
@@ -88,7 +86,7 @@ func GetCmdDistributionRecordForRecipient(queryRoute string) *cobra.Command {
 //GetCmdDistributionRecordForDistNameAll returns all records for a given distribution name
 func GetCmdDistributionRecordForDistNameAll(queryRoute string) *cobra.Command {
 	return &cobra.Command{
-		Use:   "records-by-name-all [distribution name]",
+		Use:   "records-by-name [distribution name] [status]",
```

Comment:
Needs an example to indicate what status options are valid.

Reply:
the default returns unspecified , so technically everything is valid in this case

Reply:
An example would indicate if an int or status string is expected.